packages/ui-map-components/src/components/Legend/MarkerLegend.tsx

```diff
@@ -27,16 +27,22 @@ import {
 } from '../../utils';
 import { LegendEntry } from './LegendEntry';
 
-const Container = styled(MuiBox)`
+const Container = styled(MuiBox)<{
+  $isExport?: boolean;
+}>`
   display: flex;
   align-items: center;
   justify-content: center;
   flex-wrap: wrap;
 
-  ${p => p.theme.breakpoints.down('sm')} {
-    flex-direction: column;
+  ${({ $isExport, theme }) =>
+    !$isExport &&
+    `
+  ${theme.breakpoints.down('sm')} {
+    flex-direction:  column;
```

Comment:
This one might need to go through Prettier again

```suggestion
    css`
  ${theme.breakpoints.down('sm')} {
    flex-direction: column;
```